jupyter-book/preprocessing_visualization/normalization.ipynb

```diff
@@ -1,12 +1,10 @@
 {
```

Comment:
<p>Can you add the missing TODO citation?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/single-cell-best-practices/pull/139/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>